lib/srv/db/ca.go

```diff
@@ -178,7 +195,12 @@ func (d *realDownloader) Download(ctx context.Context, database types.Database)
 	case types.DatabaseTypeCloudSQL:
 		return d.downloadForCloudSQL(ctx, database)
 	case types.DatabaseTypeAzure:
-		return d.downloadFromURL(azureCAURL)
+		if strings.HasSuffix(azureCAURLBaltimore, hint) {
```

Comment:
Don't have strong preference but we can consider returning [][]byte that will combine `azureCAURLBaltimore` and `azureCAURLDigiCert` certs and get rid of `hint string` parameter.

Reply:
ya, `hint` is definitely a small hack. was trying to avoid the case to download both cert files when one is already there, without rewriting the code too much. i'll keep it like this for now